src/components/Tooltip/Tooltip.js

```diff
@@ -447,6 +447,11 @@ class Tooltip extends Component {
           'aria-owns': tooltipId,
         };
 
+    const ariaDescribedbyProps = !open
+      ? {}
+      : {
+          'aria-describedby	': tooltipId,
```

Comment:
just wondering is there a reason for the trailing space?

Reply:
Removed the trailing space. Thanks!